src/web/src/components/Banner.tsx

```diff
@@ -4,81 +4,81 @@ import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import { telescopeUrl } from '../config';
 import BannerDynamicItems from './BannerDynamicItems';
 import ScrollAction from './ScrollAction';
+import LandingButtons from './LandingButtons';
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
-    h1: {
-      position: 'absolute',
-      color: theme.palette.text.primary,
-      fontWeight: 'bold',
-      fontSize: '12rem',
-      display: 'block',
-      top: theme.spacing(20),
-      left: theme.spacing(15),
-      [theme.breakpoints.between('xs', 'sm')]: {
-        fontSize: '4rem',
-        textAlign: 'left',
-        left: theme.spacing(4),
-        right: theme.spacing(4),
-        top: theme.spacing(14),
-      },
-      [theme.breakpoints.between('md', 'lg')]: {
-        fontSize: '8rem',
-      },
-      [theme.breakpoints.up('xl')]: {
-        fontSize: '12rem',
-      },
-    },
     heroBanner: {
       maxHeight: '100vh',
       overflow: 'hidden',
       position: 'relative',
     },
+    textsContainer: {
+      position: 'absolute',
+      width: '100%',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center',
+      top: '40%',
+    },
+    telescopeTitle: {
+      color: '#A0D1FB',
+      fontWeight: 'bold',
+      fontSize: 'clamp(2.5rem, 4.2vw, 6em)',
+      letterSpacing: '.45em',
+      display: 'block',
+      zIndex: 1000,
+      marginBottom: '10px',
+    },
+    quoteText: {
+      width: '70%',
+      color: '#FFFFFF',
```

Comment:
Let's add this colour to the theme

Reply:
I don't want to connect the landing page with the theme colors now, let's make this when implementing a dark/light themes in the landing page. For now, let's land the new land page without a dark/light theme.